test/config.sh

```diff
@@ -284,7 +284,9 @@ globalExcludeTests+=(
 	[nats-streaming_utc]=1
 	[nats_no-hard-coded-passwords]=1
 	[nats_utc]=1
+	[swarm_cve-2014--shellshock]=1
```

Comment:
I didn't find a mention of "how to exclude an image for everything"; perhaps it's there but I didn't find it

Reply:
With no tags in the library file, there is nothing to test so any mentions of swarm can just be removed.

(The failed GitHub actions email you are getting is because it is generating an empty matrix to run tests on and that is fine)

Reply:
okay; I'll remove these lines: thanks!